README.md

```diff
@@ -35,6 +35,13 @@ standards, e.g., ISO 26262.
 After you cloned this repository into your ROS 2 workspace folder, you may build and install it using colcon:
 $ `colcon build --packages-select-regex micro_ros_.*diagn`
 
+In a typical architecture indicated above, you will need to build the packages 
+[micro_ros_diagnostic_msgs](./micro_ros_diagnostic_msgs/),
+[micro_ros_diagnostic_updater](./micro_ros_diagnostic_updater/), and
+[micro_ros_common_diagnostics](./micro_ros_common_diagnostics/) on the microcontroller. Build the packages
```

Comment:
Right now, common diagnostics has a running example hwmonitor. If the common diagnostics is intended to be built on the microcontroller, the example should be disabled.  I don't think we should enforce the user to use the provided monitors